idsvr/templates/pvc.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if and .Values.curity.config.persistentConfigVolume.enabled (not .Values.curity.config.persistentConfigVolume.existingClaim) }}
```

Comment:
I think this should run regardless of whether `existingClaim` is set to true, so that the desired state:

- This recovers if for example the pvc gets deleted. 
- Otherwise it is a no-op on the next deployment.
- Also, I think it will enable the use of persistent volumes to survive events like cluster deletion (if the customer backs up the persistent volume yaml).